exchange/price_granularity_test.go

```diff
@@ -0,0 +1,35 @@
+package exchange
+
+import (
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"testing"
+)
+
+func TestGetPriceBucketString(t *testing.T) {
+	price := 1.87
+	getOnePriceBucket(t, "low", price, "1.50")
```

Comment:
Seems like you might need to add a test here for `precision: 0` too? Assuming that line I called out in the main file is actually buggy, of course.